react_native_fundamentals/presentation.html

```diff
@@ -57,17 +60,26 @@ <h4 class="slide-name">AGENDA</h4>
             <li class="fragment">Architecture</li>
             <li class="fragment">Code Quality Tools</li>
           </ul>
+          <aside class="notes">
+            <strong>1. Introduction:</strong> This section provides an introduction to the topic by covering an overview of iOS and Android platforms, an explanation of React Native as a framework for cross-platform development and a mention of alternative options.<br>
+            <strong>2. Environment setup:</strong> This part focuses on the necessary steps to set up the development environment for React Native. <br>
+            <strong>3. Project structure overview:</strong>  This section provides an overview of the project's structure, including the organization of files and folders. <br>
+            <strong>4. Components:</strong> This section covers the core components in React Native, explaining their purpose and usage.<br>
+            <strong>5. Styles:</strong> This section is dedicated to discussing styling in React Native. It covers the use of the StyleSheet API for defining and managing styles in a React Native app.<br>
+            <strong>6. Architecture:</strong> This section delves into the old and new architectural approaches in React Native, highlighting key concepts.<br>
+            <strong>7. Code Quality Tools:</strong> This section focuses on code quality tools used in React Native development. It may discuss the use of ESLint and Prettier for code formatting and consistency and mention Husky as a tool for integrating pre-commit hooks. <br>
+          </aside>  
         </section>
         
-        <section class="topic-name-slide">
+        <!-- <section class="topic-name-slide">
           <h4>Introduction</h4>
-        </section>
+        </section> -->
 
         <section class="picture-slide">
           <h4 class="slide-name">What is React Native</h4>
           <img src="assets/images/intro.png" alt="simple" />
           <aside class="notes">
-            React Native is a popular <strong>JavaScript-based mobile app framework</strong> that allows you to build natively-rendered mobile apps for iOS and Android. It also can be used to develop applications for Web. The framework lets you create an application for various platforms by using the same codebase. React Native also allows you to implement code in OS-native languages.<br>
+            React Native is a popular <strong>JavaScript-based mobile app framework</strong> that allows you to build natively-rendered mobile apps for iOS and Android. It also can be used to develop applications for Web. The framework lets you create an application for various platforms by using the same codebase. React Native also allows you to implement code in OS-native languages: Swift and Objective-C for iOS, and Java and Kotlin for Android.<br>
```

Comment:
The framework lets you create an application for various platforms by using the same codebase. ->
The framework lets you create **applications** for various platforms by using the same codebase.

Reply:
Done